creator/ingest_runs/tasks/validation_run.py

```diff
@@ -249,15 +248,17 @@ def run_validation(validation_run_uuid=None):
     )
     vr.start()
     vr.save()
+
     try:
         results = validate_file_versions(vr)
         report_markdown = build_report(results)
         vrs = persist_results(results, report_markdown, vr)
-    except Exception:
+    except Exception as e:
         vr.success = False
-        vr.fail()
+        vr.fail(error_msg=str(e))
         vr.save()
-        raise
+        er_msg = "Error occurred during validation run process."
```

Comment:
Isn't this going to overwrite whatever message is in `e`? I think you just need to replace 

```python
er_msg = "Error occurred during validation run process."
raise type(e)(er_msg).with_traceback(sys.exc_info()[2])
```
with `raise`